externs/shaka/player.js

```diff
@@ -928,6 +929,12 @@ shaka.extern.InitDataTransform;
  *   Might result in undesirable behavior if mediaSource.codecSwitchingStrategy
  *   is not set to SMOOTH.
  *   Defaults to true if SMOOTH codec switching is supported, RELOAD overwise.
+ * @property {boolean} useStreamOnceInPeriodFlattening
+ *   If period combiner is used, this option ensures every stream is used
+ *   only once in period flattening. It speeds up underlying algorithm
+ *   but may raise issues if manifest does not have stream consistency
+ *   between periods.
+ *   Defaults to <code>false</code>.
```

Comment:
For me this should default to true, @littlespex , what do you think?

Reply:
I'd be more than happy to set it to `true` by default, but potentially it will increase number of `PERIOD_FLATTENING_FAILED` errors on not well structured manifests. We'd have to add mention of this setting in error description or in FAQ (or both).

Reply:
Defaulting to `true` technically be a breaking change for us. You would have call out in the release notes that if you must opt-out if you want consistent behavior with previous versions after upgrading.